entity-management-solr/src/main/java/eu/europeana/entitymanagement/solr/model/SolrEntity.java

```diff
@@ -0,0 +1,142 @@
+package eu.europeana.entitymanagement.solr.model;
+
+import eu.europeana.entitymanagement.definitions.model.Entity;
+import eu.europeana.entitymanagement.vocabulary.AgentSolrFields;
+import eu.europeana.entitymanagement.vocabulary.EntitySolrFields;
+import org.apache.solr.client.solrj.beans.Field;
+
+import java.util.List;
+import java.util.Map;
+
+public abstract class SolrEntity<T extends Entity> {
+
+    @Field(AgentSolrFields.TYPE)
+    private String type;
+
+    @Field(AgentSolrFields.ID)
```

Comment:
This is a common field, please use the EntitySolrFields class insted

Reply:
Done.